gen/gen.c

```diff
@@ -138,6 +138,13 @@ u_int opt_nflow = 0;
 
 bool use_curses = true;
 
+/* Some time related parameters for RFC2544 tests in seconds . */
+#define RFC2544_WARMUP_SECS		3	/* to keep RFC2544_WARMUP */
```

Comment:
I think that the comment `to keep RFC2544_WARMUP` is unnecessary.  It first confused me, though I eventually understood what it meant.

Ditto for the following three lines.

Reply:
You mean only keep the comment of RFC2544_TRIAL_SECS_DEFAULT or remove all of five comments?

Reply:
I mean the former.

Reply:
Done.